Project/Assets/Prairie/Framework/Script/Interaction/AssociatedTwineNodes.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections;
 using System.Collections.Generic;
 
+[AddComponentMenu("Prairie/Twine/Associated Twine Nodes")]
```

Comment:
I think this is the right submenu for `AssociatedTwineNode`, which is sorta different from all other interactions.

My question is what menu does `TwineNode` go under (currently unassigned a menu)? We don't typically have folks manually add them (and we'd actually like to avoid that implication) so I like how it isn't here in "_Prairie/Twine/_" - but this also feels incomplete without including it somewhere.

Maybe a _Prairie/Support_ or _Prairie/Utility_? A folder that says "hey, these are important to Prairie, but they are used by the Prairie system, not by you directly"